### PR TITLE
 CSS clean up of privacy page

### PR DESCRIPTION
**Pull request by Davidlloyd8**

### CODE REVIEW

**Comment by Davidlloyd8** [comment]
@a11rew

**Comment by a11rew** [comment]
What's with the number of commits?

**Comment by Davidlloyd8** [comment]
When i pull 
i accepted incomming changes
that why that commit where much 
don't really know the reason though

**Comment by Davidlloyd8** [comment]
@a11rew

File: packages/control/package.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
     "identity-obj-proxy": "^3.0.0",
     "prettier": "^2.3.2",
     "react-is": "^17.0.2",
+    "redux-logger": "^3.0.6",
```
Comment (a11rew) [inline packages/control/package.json:39]: Did you mean to make these changes?

**Comment by a11rew** [comment]
@Davidlloyd8 undo your commits, stash your changes and reset your local branch like it says in this [S/O answer](https://stackoverflow.com/a/6229837/17144515). Apply your stashed changes and make a pr, you should only have your changes in there

**Comment by eni4sure** [comment]
@Davidlloyd8 checkout the instructions on how to avoid this number of commits
I left the process on slack

**Comment by eni4sure** [comment]
@a11rew you can close this now and re-open once that is done

**Comment by Davidlloyd8** [comment]
Sorry i miss that @eni4sure can i see how to solve that

**Pull request closed without merging**